### DISCUSSION TITLE
How do I purchase Serverless Functions separately?

### DISCUSSION THREAD

**Discussion by LiJunYi2** [vercel/community#6681] [Help]
### Summary

I would like to purchase Serverless Functions separately, but I didn't find a place to purchase it on the official website? Do you offer separate purchase? thanks

### Steps to Reproduce

I would like to purchase Serverless Functions separately, but I didn't find a place to purchase it on the official website? Do you offer separate purchase?  thanks

**Comment by amyegan** [comment]
Hey @LiJunYi2. If you need higher Serverless Function usage than the [limits](https://vercel.com/docs/limits/overview) of Hobby plan allow, you can upgrade to the [Pro plan](https://vercel.com/docs/accounts/plans/pro#) which has higher limits. 

An "**Upgrade to Pro**" button can be found on the billing page in the Vercel dashboard, or you can click the "**Upgrade now**" button on the [pricing page](https://vercel.com/pricing).

https://vercel.com/docs/accounts/plans/pro#upgrading-to-pro

**Reply by amyegan** [reply]
If you're on a Pro plan, then you can pay for [additional usage](https://vercel.com/docs/limits/overview#additional-resources) as needed
